### PR TITLE
adding april 2024 community call blog post

### PR DESCRIPTION
**Pull request by laiyi-ohlsen**
http://website.mlab-sandbox.measurementlab.net/blog/apr24-community-call/#measuring-internet-quality-from-the-user's-perspective

This change is [Reviewable](https://reviewable.io/reviews/m-lab/website/797)

### CODE REVIEW

File: _posts/blog/2024-03-25-apr24-community-call.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: blog
+title: "Measuring Internet Quality from the User's Perspective"
```
Comment (georgiamoon) [inline _posts/blog/2024-03-25-apr24-community-call.md:3]: Measuring the Quality of an Individual's Connection

File: _posts/blog/2024-03-25-apr24-community-call.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+date: 2024-03-26
+breadcrumb: blog
+categories:
+  - community
+  - internet quality
+  - research
+---
+Interested in metrics that measure an Internet connection's quality from the end user perspective? Join us at our April M-Lab Community Call on April 4, 2042 at 11am EDT. <!--more-->
+
+## Metrics for Measuring Your Internet's Quality
```
Comment (georgiamoon) [inline _posts/blog/2024-03-25-apr24-community-call.md:14]: Metrics for Measuring the Quality of an Individual Connection

File: _posts/blog/2024-03-25-apr24-community-call.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Join us to learn more about these efforts and consider how they could, if collected at scale, be aggregated and used to assess the quality and gaps of a region's Internet quality.
+
+## Feedback
+Want to hear about a metric not listed above? [Let us know](hello@measurementlab.net)!
+
+## Registration
+M-Lab's Community Calls take place on the first Thursday of every month at 11am Eastern Time. Please [register via Zoom](https://us02web.zoom.us/meeting/register/tZwucuCgqTIiHdd1OVBDlvHVHKGfyGGoMn6U) and reach out to [hello@measurementlab.net](mailto:hello@measurementlab.net) with any questions.
+
+## Support
+Internet Quality Barometer is generously supported by an [Internet Society Research Grant](https://www.isocfoundation.org/2024/01/announcing-our-2023-research-grantee-cohort-14-projects-examining-the-future-of-the-internet/).
```
Comment (georgiamoon) [inline _posts/blog/2024-03-25-apr24-community-call.md:33]: "Internet Society Foundation Research Grant"

**Review by georgiamoon — CHANGES_REQUESTED** [review]
Made some suggested edits, but after that LGTM

**Review by laiyi-ohlsen — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/m-lab/website/797)* status: 1 change requests, 0 of 1 approvals obtained (waiting on @georgiamoon)

___
*[`_posts/blog/2024-03-25-apr24-community-call.md` line 3 at r1](https://reviewable.io/reviews/m-lab/website/797#gh-1541991121:-Nu90QLD5sbd38jbfNGs:b-896fix) ([raw file](https://github.com/m-lab/website/blob/aa17d1a0ffbfe029b7dad0bce2c17d5fe3044803/_posts/blog/2024-03-25-apr24-community-call.md#L3)):*
<details><summary><i>Previously, georgiamoon (georgia bullen) wrote…</i></summary><blockquote>

Measuring the Quality of an Individual's Connection
</blockquote></details>

Done.
___
*[`_posts/blog/2024-03-25-apr24-community-call.md` line 14 at r1](https://reviewable.io/reviews/m-lab/website/797#gh-1541991699:-Nu90QnwCwscvSlik0Y1:b-896fix) ([raw file](https://github.com/m-lab/website/blob/aa17d1a0ffbfe029b7dad0bce2c17d5fe3044803/_posts/blog/2024-03-25-apr24-community-call.md#L14)):*
<details><summary><i>Previously, georgiamoon (georgia bullen) wrote…</i></summary><blockquote>

Metrics for Measuring the Quality of an Individual Connection
</blockquote></details>

Done.
___
*[`_posts/blog/2024-03-25-apr24-community-call.md` line 33 at r1](https://reviewable.io/reviews/m-lab/website/797#gh-1541992873:-Nu90RLr8avR8NJlyy1N:b-896fix) ([raw file](https://github.com/m-lab/website/blob/aa17d1a0ffbfe029b7dad0bce2c17d5fe3044803/_posts/blog/2024-03-25-apr24-community-call.md#L33)):*
<details><summary><i>Previously, georgiamoon (georgia bullen) wrote…</i></summary><blockquote>

"Internet Society Foundation Research Grant"
</blockquote></details>

Done.

**Review by laiyi-ohlsen — COMMENTED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/m-lab/website/797)* status: 1 change requests, 0 of 1 approvals obtained (waiting on @georgiamoon)

**Review by georgiamoon — APPROVED** [review]

**Pull request merged**